### PR TITLE
Update spec for new path for restart service endpoint

### PR DESCRIPTION
**Pull request by kbatuigas**
[Deploy preview](https://deploy-preview-432--redpanda-docs-preview.netlify.app/api/admin-api.html#put-/debug/restart_service)

- Changes path from `/redpanda-services/restart` to `/debug/restart_service`
- Also adds endpoint under "Debugging" tag
- Add accepted values for parameter

Fixes https://github.com/redpanda-data/documentation-private/issues/2283

### CODE REVIEW

File: modules/ROOT/attachments/admin-api.yaml [hunk 1]
```diff
@@ -1463,6 +1463,29 @@ paths:
...
+      summary: Restart service
+      description: Restart a Redpanda service.
+      operationId: restart_service
+      parameters:
+        - name: service
+          in: query
+          required: true
+          schema:
+            type: string
+          example: 'schema-registry'
```
Comment (kbatuigas) [inline modules/ROOT/attachments/admin-api.yaml]: @BenPope Do we have a list of acceptable values for `service=`?

Comment (BenPope) [reply modules/ROOT/attachments/admin-api.yaml]: Just these two: https://github.com/redpanda-data/redpanda/blob/v23.3.12/src/v/redpanda/admin/server.cc#L4318-L4321

**Review by BenPope — APPROVED** [review]
LGTM

**Review by Feediver1 — APPROVED** [review]
LGTM!

File: modules/ROOT/attachments/admin-api.yaml [hunk 2]
```diff
@@ -1476,11 +1476,9 @@ paths:
...
           schema:
             type: string
-          examples:
-            schema-registry:
-              value: 'schema-registry'
-            http-proxy:
-              value: 'http-proxy'
+            enum: 
+            - http-proxy
+            - schema-registry
```
Comment (BenPope) [inline modules/ROOT/attachments/admin-api.yaml:1481]: :fire:

**Review by BenPope — APPROVED** [review]

**Review by Deflaimun — APPROVED** [review]

**Pull request merged**